test/router_test.exs

```diff
@@ -102,6 +102,33 @@ defmodule BeeBee.RouterTest do
     end
   end
 
+  describe "DELETE /banana" do
+    test "deletes the 'banana' short tag" do
+      _conn =
+        :post
+        |> conn("/_add", %{"url" => "https://github.com", "short_tag" => "banana"})
+        |> Router.call([])
+
+      conn =
+        :delete
+        |> conn("/banana")
+        |> Router.call([])
+
+      assert conn.status == 202
+      assert %{} = Jason.decode!(conn.resp_body)
+    end
+
+    test "fails for non-existent short tag" do
+      conn =
+        :delete
+        |> conn("/banana")
+        |> Router.call([])
+
+      assert conn.status == 422
+      assert %{"errors" => ["Short tag not found"]} = Jason.decode!(conn.resp_body)
+    end
+  end
+
```

Comment:
Great tests! There should also be some to make sure authentication is enabled for this endpoint, see the section below:
```
  describe "Basic Authentication" do
```